src/utils.tsx

```diff
@@ -78,6 +78,10 @@ export const trimStringValues = (object: any): any => {
 
 export const formatDateTimeStrings = (dateTime: string): string => {
   const date = parseISO(dateTime);
-  const formattedDate = format(date, 'dd MMM yyyy HH:mm');
+  const formattedDate =
```

Comment:
Can you change this function to have two variables `(date: String , includesTime (or withTime):boolean)`. and use the includesTime (or withTime) variable for the formatting.

If you look at the snapshots some of the created times does not include the time with the current implementation

Reply:
![image](https://github.com/ral-facilities/inventory-management-system/assets/83226114/437b9028-e895-441a-baea-5177e02cfb82)